stream_django/enrich.py

```diff
@@ -118,7 +119,16 @@ def _inject_objects(self, activities, objects, fields):
             if not self.is_ref(activity, field):
                 continue
             f_ct, f_id = activity[field].split(':')
-            instance = objects[f_ct].get(int(f_id))
+
+            try:
+                f_id = int(f_id)
+            except ValueError:
+                try:
+                    f_id = uuid.UUID(f_id)
```

Comment:
you can probably get the right fn to use here from Django (eg. lookup the PK for the model and use the `to_python` method)